src/azure-cli/azure/cli/command_modules/resource/tests/hybrid_2018_03_01/test_resource_custom.py

```diff
@@ -325,8 +325,7 @@ def test_deployment_prompt_file_order(self):
         result_parameters = _process_parameters(template, parameter_list)
         missing_parameters = _find_missing_parameters(result_parameters, template)
 
-        param_file_order = ["[u'secureParam', u'boolParam', u'enumParam', u'arrayParam', u'objectParam']",
-                            "['secureParam', 'boolParam', 'enumParam', 'arrayParam', 'objectParam']"]
+        param_file_order = ["['secureParam', 'boolParam', 'enumParam', 'arrayParam', 'objectParam']"]
```

Comment:
There is no need to put `"['secureParam', 'boolParam', 'enumParam', 'arrayParam', 'objectParam']"` in a list anymore.

L330 should also use 

```py
self.assertTrue(str(list(results.keys())) == param_file_order)
```

@zhoxing-ms